src/drivers/wdm/queries/ExaminedValue/ExaminedValue.ql

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/**
+ * @name ExaminedValue
+ * @kind problem
+ * @description The returned value is annotated with the _Check_return_ or _Must_inspect_result_ annotation, but the calling function is either not using the value or is overwriting the value without examining it.

```

Comment:
Please add a reference to the ID of the Code Analysis check this is a port of (C28193)

Reply:
reference added. There is also a reference in the .qhelp file. I've noticed that's how other CodeQL queries add documentation.

Reply:
Yep, the reference there is good.  I should've been more clear that I'd just like it if the ID is mentioned in the description here, not necessarily a full MSDN link.  But not gonna block checkin on that.